packages/terra-clinical-data-grid/src/DataGrid.module.scss

```diff
@@ -122,4 +122,14 @@
       flex: 1 1 auto;
     }
   }
+
+  .hidden-info {
```

Comment:
I wasn't sure on the naming of this className for the hidden label/description. Open to suggestions

Reply:
Would Terra VisuallyHiddenText be sufficient for this scenario? We also added something similar to slide panel https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/visually-hidden-text/about

Reply:
I will try using that. That seems like what we'd want to use here if it means we can avoid adding custom css

Reply:
Addressed here: https://github.com/cerner/terra-clinical/commit/b2683006e880a6d9b60d7939bd96273b829a3f80